src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+public interface AreaCalculator {
+    public abstract double getArea();
```

Comment:
Remove unnecessary 'public abstract' keywords, as they are implicit in interface methods. Simply use 'double getArea();'